pkgs/development/python-modules/natasha/default.nix

```diff
@@ -0,0 +1,32 @@
+{ buildPythonPackage, fetchPypi, python, pymorphy2, razdel, navec, slovnet
+, yagry, ipymarkup, pytest, isPy3k, lib }:
+buildPythonPackage rec {
+  pname = "natasha";
+  version = "1.6.0";
+  format = "setuptools";
+
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-Rgguazgq06a8B9jeRnfHD5VTR+Xrd+8OCsQUfaGLEq0=";
+  };
+
+  propagatedBuildInputs = [ pymorphy2 navec razdel slovnet yagry ipymarkup ];
+
+  nativeCheckInputs = [ pytest ];
+
+  checkPhase = ''
+    ${python.interpreter} -m pytest tests
+  '';
+
+  pythonImportCheck = [ "natasha" ];
+
+  meta = with lib; {
+    description =
+      "Natasha solves basic NLP tasks for Russian language: tokenization, sentence segmentation, word embedding, morphology tagging, lemmatization, phrase normalization, syntax parsing, NER tagging, fact extraction";
```

Comment:
This sentence is a bit long for description, IMO.
How about citing `Solves basic Russian NLP tasks, API for lower level Natasha projects` from GitHub or `Named-entity recognition for russian language`?
https://github.com/natasha/natasha/blob/ebe33ff1be333af0059aa34da825dfd4abb15dc2/setup.py#L13

Reply:
I shortened the description.